src/libs/parsers/formatDates.ts

```diff
@@ -31,11 +32,15 @@ export const decomposeDate = (timestamp: number) => {
 
 // a-t-on encore besoin du type Date ?
 export const formatToFrenchDate = (
-  date: Date | number | string | { day: number; month: string; year: number }
+  date: Date | number | string | { day: number; month: string; year: number },
+  shouldFormatFirstDayOfTheMonth?: boolean
 ) => {
   const formatedDate = typeof date === 'string' ? new Date(date) : date
   const timestamp = formatedDate instanceof Date ? formatedDate.getTime() : formatedDate
   const { day, month, year } = typeof timestamp === 'number' ? decomposeDate(timestamp) : timestamp
+
+  if (shouldFormatFirstDayOfTheMonth) return `${day}er ${month} ${year}`
```

Comment:
```suggestion
  if (isFirstDayOfMonth(releaseDate)) return `${day}er ${month} ${year}`
```
(low logaf)
Et du coup supprimer le paramètre de la fonction. Ça simplifie un peu le code.

Reply:
Ah oui ... Je comprends. Mais c'est vrai que je vois pas de cas où on veut écrire "1 mai 2024"

Reply:
regarde sur le screen, y a cet exemple justement, c'est le cas actuel en fait, mais je peux demander à Aline si elle veut que ça change sur toute l'app et si oui, je pourrais retirer la condition

Reply:
Vu avec Aline, on garde ce change pour les séances cinés pour le moment